core/protocol/protocol.go

```diff
@@ -43,6 +46,7 @@ var bnStrings = map[Network]string{
 	MainNet:  "MainNet",
 	TestNet:  "TestNet",
 	PrivNet:  "PirvNet",
+	TestPowNet:  "TestPowNet",
```

Comment:
don't need the additional config profile since we can change the test-net from the pure blake2b to the mix mode or the pure cuckoo mode.